Assets/MixedRealityToolkit/_Core/Inspectors/PropertyDrawers/Vector3RangePropertyDrawer.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Attributes;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Inspectors.PropertyDrawers
+{
+    [CustomPropertyDrawer(typeof(Vector3RangeAttribute))]
+    public class Vector3RangePropertyDrawer : PropertyDrawer
+    {
+        public override float GetPropertyHeight(SerializedProperty property, GUIContent label)
```

Comment:
Summary Tag

Reply:
Why? It's a unity override method. I thought we were not documenting unity types/methods.